subprojects/module-test/module-test.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'java'
+}
+
+if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
+    project.tasks.all { task -> task.enabled = false }
```

Comment:
You could simplify it with [`onlyIf`](https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:using_a_predicate)

Reply:
To have just one line.